enterprise/frontend/test/metabase-enterprise/sandboxes/sandboxes.cy.spec.js

```diff
@@ -825,6 +826,117 @@ describeWithToken("formatting > sandboxes", () => {
       });
       cy.contains("37.65");
     });
+
+    it("attempt to sandbox based on question with more columns than a sandboxed table should provide meaningful UI error (metabase#14612)", () => {
+      const [ORDERS_ALIAS, PRODUCTS_ALIAS, REVIEWS_ALIAS] = [
+        "Orders",
+        "Products",
+        "Reviews",
+      ];
+      const QUESTION_NAME = "Extra columns question";
+      const ERROR_MESSAGE =
+        "Sandbox Cards can't return columns that aren't present in the Table they are sandboxing.";
```

Comment:
Might be worth reworking his test to check the *other* error case (about changing types)... otherwise this test is going to have to be rewritten once we merge in #14675